eq-author/src/App/settings/ThemesPage.js

```diff
@@ -137,32 +210,40 @@ const ThemesPage = ({ questionnaire }) => {
                       />
                     </Field>
                     <HorizontalSeparator />
-                    {themes.map(({ title, defaultOpen }) => (
-                      <CollapsibleToggled
-                        key={`${title}-toggle`}
-                        title={title}
-                        defaultOpen={defaultOpen}
-                      >
-                        {/* Added some filler text to demonstrate the opening and 
+                    {matchThemes(themes, questionnaireThemes).map(
+                      ({ shortName, title, eqId, enabled }) => (
+                        <CollapsibleToggled
+                          key={`${title}-toggle`}
+                          title={title}
+                          defaultOpen={enabled}
+                          questionnaireId={id}
+                          shortName={shortName}
+                          onChange={() => toggleTheme({ shortName, enabled })}
+                          data-test={`${shortName}-toggle`}
+                        >
+                          {/* Added some filler text to demonstrate the opening and 
                             closing; this should be removed in future tickets where 
                             we add the actual functionality. 
                           */}
-                        <p>
-                          Lorem ipsum dolor sit amet, consectetur adipiscing
-                          elit. Praesent ut eros a turpis tincidunt consectetur
-                          sit amet quis enim. Vivamus scelerisque finibus erat
-                          id mattis. In leo dolor, faucibus non volutpat vel,
-                          pellentesque et nibh.
-                        </p>
-                        <p>
-                          Phasellus viverra malesuada tincidunt. Fusce vulputate
-                          odio mauris, eu finibus nisl luctus quis. Sed
-                          dignissim dapibus sapien, at sollicitudin neque auctor
-                          non. Interdum et malesuada fames ac ante ipsum primis
-                          in faucibus.
-                        </p>
-                      </CollapsibleToggled>
-                    ))}
+                          <p />
+                          <Field>
+                            <Label>EQ ID</Label>
+                          </Field>
+                          <EqIdInput
+                            eqId={eqId}
+                            shortName={shortName}
+                            handleBlur={handleEQIdBlur}
```

Comment:
```suggestion
```

If you implement the above suggestion of moving the onBlur handler into the `EqIdInput` component, you can remove this line.